lib/rufus/lua/lib.rb

```diff
@@ -114,6 +137,10 @@ def self.path
     callback :cfunction, [ :pointer ], :int
     attach_function :lua_pushcclosure, [ :pointer, :cfunction, :int ], :void
     attach_function :lua_setfield, [ :pointer, :int, :string ], :void
+
+    include lua_version_specific_funcs
+
+    LUA_REGISTRY_INDEX = lua_version_specific_funcs::LUA_REGISTRYINDEX
```

Comment:
The goal here is to have the actual Rufus::Lua code not care which Lua version is getting used.